src/profile/data/pact-profile.test.js

```diff
@@ -0,0 +1,77 @@
+import path from 'path';
```

Comment:
I think it would be good to add a file-level comment explaining that this isn't actually a test, that it just generates the contract files.